python/sbp/client/handler.py

```diff
@@ -167,6 +60,38 @@ def __enter__(self):
   def __exit__(self, *args):
     self.stop()
 
+  # This exception is raised when a message is dispatched to a garbage
+  # collected upstream iterator.
+  class _DeadCallbackException(Exception): pass
```

Comment:
Sorry, I meant to say "nested class". The alternative would be to have it defined, not nested, at the top of the file with an underscored name. I don't think nested classes are that common. I think any caller outside this function will have to reference both the module path and the class to get at the exception.